packages/common/src/TransactionUtils.ts

```diff
@@ -188,23 +188,40 @@ export const blockUntilBlockMined = async (web3: Web3, blockerBlockNumber: numbe
     await new Promise((r) => setTimeout(r, delay));
   }
 };
+
+/**
+ * @notice Finds block closest to target timestamp. User can configure search based on error tolerance.
+ * @param web3 Web3 network to search blocks on.
+ * @param targetTimestamp Timestamp that we are finding a block for.
+ * @param higherLimitMax Returned block must have timestamp less than targetTimestamp + higherLimitMax. Increasing this
+ * increases error and reduces time to compute.
+ * @param lowerLimitMax Returned block must have timestamp more than targetTimestamp - lowerLimitMax. Increasing this
+ * increases error and reduces time to compute.
+ * @param blockDelta Amount of blocks to hop when binary searching for block. Increasing this increases error
+ * but significantly reduces time to compute.
+ * @param averageBlockTime Decreasing average block size will decrease precision and also decrease the amount of
+ * requests made in order to find the closest block. Increasing this reduces time to compute but increases requests.
+ * @returns {number, number} Block height and difference between block timestamp and target timestamp
+ */
 export async function findBlockNumberAtTimestamp(
   web3: Web3,
   targetTimestamp: number,
   higherLimitMax = 15,
-  lowerLimitMax = 15
+  lowerLimitMax = 15,
+  blockDelta = 4,
```

Comment:
Why not default this to 1 to keep previous behavior the same

Reply:
Good point will change this and the 13